frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { useQuery } from "@apollo/client";
+
+import "./TranslationPage.css";
+import { useParams } from "react-router-dom";
+import TranslationProgressBar from "../translation/TranslationProgressBar";
+import TranslationTable from "../translation/TranslationTable";
+import { StoryLine } from "../translation/Autosave";
+import { GET_STORY_AND_TRANSLATION_CONTENTS } from "../../APIClients/queries/StoryQueries";
+import CommentsPanel from "../review/CommentsPanel";
+
+type ReviewPageProps = {
+  storyIdParam: string | undefined;
+  storyTranslationIdParam: string | undefined;
+};
+
+type Content = {
+  id: number;
+  lineIndex: number;
+  content: string;
+  status: string;
+};
+
+const ReviewPage = () => {
+  const {
+    storyIdParam,
+    storyTranslationIdParam,
+  } = useParams<ReviewPageProps>();
+
+  const storyId = +storyIdParam!!;
+  const storyTranslationId = +storyTranslationIdParam!!;
+  const [translatedStoryLines, setTranslatedStoryLines] = useState<StoryLine[]>(
+    [],
+  );
+  const [numTranslatedLines, setNumTranslatedLines] = useState(0);
+
+  useQuery(GET_STORY_AND_TRANSLATION_CONTENTS(storyId, storyTranslationId), {
+    fetchPolicy: "cache-and-network",
+    onCompleted: (data) => {
+      const storyContent = data.storyById.contents;
+      const translatedContent = data.storyTranslationById.translationContents;
+
+      setNumTranslatedLines(data.storyTranslationById.numTranslatedLines);
+
+      const contentArray: StoryLine[] = [];
+      storyContent.forEach(({ content, lineIndex }: Content) => {
+        contentArray.push({
+          lineIndex,
+          originalContent: content,
+          status: "Default",
+        });
+      });
+
+      contentArray.sort((a, b) => a.lineIndex - b.lineIndex);
+
+      translatedContent.forEach(({ id, content, lineIndex }: Content) => {
+        const arrIndex = lineIndex - contentArray[0].lineIndex;
+        contentArray[arrIndex].translatedContent = content;
+        contentArray[arrIndex].storyTranslationContentId = id;
+      });
+      setTranslatedStoryLines(contentArray);
+    },
+  });
+
+  return (
+    <div className="translation-page">
+      <h1>Story Title Here</h1>
```

Comment:
Going to leave this as a separate task since theres a number of miscellaneous story data that needs to be displayed  (language, youtube links, etc)
[new task](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=4a47edc475b34bb99b25ff1fae2a5507&p=a7f925709bda41d3985e52de7425dc8d)